_posts/2021-05-22-fluxv2-openfaas.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "See how Flux's Git Ops ToolKit can simplify delivery"
+description: "See how Flux can simplify serverless deployments to OpenFaaS"
+date: 2021-05-22    
+image: /images/2021-05-22-flux/earth.jpg
+categories:
+- arkade
+- kubectl
+- gitops
+- openfaas-operator
+- flux
+- CI/CD
+author_staff_member: alistair
+dark_background: true
+
+---
+
+We learn how to manage OpenFaaS and install functions using Flux.
+
```

Comment:
In the introduction give a very short list / overview of all the things you plan to cover and what the outcome will be.

For instance:

In this tutorial we'll create a Kubernetes cluster, install Flux V2, link it to our GitHub Account and deploy OpenFaaS along with a number of functions using a GitOps approach.